### DISCUSSION TITLE
How to pass Maven mirrors?

### DISCUSSION THREAD

**Discussion by ponchofiesta** [docToolchain/docToolchain#1437] [Q&A]
We're using internal mirrors servers and don't have internet access. How to pass Repository mirrors to doctoolchain? I tried a build.gradle file but it looks like it is ignored:

```
repositories {
  maven {
    credentials {
        username System.getenv('DEVSTACK_USERNAME')
        password System.getenv('DEVSTACK_PASSWORD')
    }
    url "https://XXX/artifactory/public/"
  }
  maven {
    credentials {
        username System.getenv('DEVSTACK_USERNAME')
        password System.getenv('DEVSTACK_PASSWORD')
    }
    url "https://XXX/artifactory/gradle-plugins/"
  }
}
```

**Comment by rdmueller** [comment]
please take a look at https://doctoolchain.org/docToolchain/v2.0.x/020_tutorial/160_EnterpriseTipsAndTricks.html and see if it works for you. Feedback would be great!

**Reply by ponchofiesta** [reply]
Thank you. I'll try it.
In general I'm missing a way to provide Confluence `bearerToken` without saving in the config file. It should be possible to provide any credentials in runtime without saving to general config files.
